server/plugins/error-logger.ts

```
@@ -0,0 +1,8 @@
+export default defineNitroPlugin((nitroApp) => {
+  nitroApp.hooks.hook('error', async (error, { event }) => {
```

ESLint

[error] 'event' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)